### PR TITLE
Contact page added

### LINKED ISSUE #68: Add the Contact page
- self-explanatory

### PR DESCRIPTION
**Pull request by BISHALTWR**
- Added Contact Us page
- Closes #68 

Full screen

half_screen_LOL

filled_form

 > Send button just logs the content of form
 
> `Contact Us` feels like a better header than `Contact Details`

### CODE REVIEW

**Review by gaurovgiri — CHANGES_REQUESTED** [review]
Screenshot 2024-10-09 at 10 31 33 PM

Can you center align the `send` button and `follow us` part for small screens. Everything else looks great. I'm dubious about using the layout as default here. But it's fine for now.

**Comment by BISHALTWR** [comment]
- Update:
small_screen

File: contact.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          <input type="text" id="subject" class="w-full h-[56px] bg-[#D9D9D9] rounded-md border-none focus:outline-none pl-2" placeholder="" required />
+        </div>
+
+        <div class="flex flex-col items-start gap-1 w-full">
+          <label class="font-inter font-bold text-lg text-black">Message</label>
+          <textarea id="message" class="w-full h-[166px] bg-[#D9D9D9] rounded-md border-none focus:outline-none pl-2 pt-2 pr-2 resize-none" placeholder="" required></textarea>
+        </div>
+
+        <div class="w-full flex justify-center lg:justify-start">
+          <button type="submit" class="flex flex-row justify-center items-center p-2 bg-[#FB8500] rounded-md w-auto transform transition-transform duration-200 active:scale-95">
```
Comment (aayush105) [inline contact.markdown:39]: Add a hover bg-color to bg-[#FB8500]/80.

**Review by aayush105 — CHANGES_REQUESTED** [review]
Instead of adding the hex value of the color directly, use the color name for respective hex color code which is predefined in the _config.yml file.

File: contact.markdown (hunk 1 above)
Comment (BISHALTWR) [reply contact.markdown:39]: Alright, will look into it.

Comment (BISHALTWR) [reply contact.markdown]: done

**Review by rpandox — APPROVED** [review]
LGTM!!!

**Review by gaurovgiri — APPROVED** [review]
AWESOME!

**Comment by prabhavpokharel** [comment]
Nice!

**Review by aayush105 — APPROVED** [review]
LGTM!!

**Pull request merged**